packages/fxa-graphql-api/src/config.ts

```diff
@@ -44,7 +44,19 @@ const conf = convict({
   contentfulApiKey: {
     doc: 'GraphQL Content API key for Contentful hCMS to fetch RP-provided content',
     format: String,
-    env: 'CONTENTFUL_API_KEY',
+    env: 'CONTENTFUL_GRAPHQL_API_KEY',
+    default: '',
+  },
+  contentfulSpaceId: {
+    doc: 'Alphanumeric id used for instantiating the ContentfulClient',
```

Comment:
Might be worth linking to Contentful's [docs on what Spaces](https://www.contentful.com/developers/docs/references/content-management-api/#/reference/spaces) are?